src/vivarium/framework/values.py

```diff
@@ -173,7 +209,7 @@ class Pipeline:
     generic components that create a set of pipeline modifiers for
     values that won't be used in the particular simulation.
 
-    Attributes
+    Parameters
```

Comment:
nit: I think it should be attributes when you're defining the _class_ docstring (but then they'd be Parameters if you were doing this in the init). Like, you're literally saying this class has the attributes ... (public only though, so really it shouldin't include source, combiner, or manager since those are all private). This all becomes tricky in vivarium w/ how many of our attrs are defined in non-init methods and so it's difficult to document.

Reply:
I normally wouldn't mess with it, but "Attributes" instead of "Parameters" after the subsequent changes caused a doc build error